src/pybind/mgr/cephadm/utils.py

```diff
@@ -13,3 +13,18 @@ def name_to_config_section(name):
         return name
     else:
         return 'mon'
+
+
+def name_to_auth_entity(name) -> str:
```

Comment:
The usage of `name_to_auth_entity` and `name_to_config_section` are somewhat confusing.

It would be nice to consolidate and/or better document their intended purpose.

Reply:
yeah, right